examples/async/index.js

```diff
@@ -1,15 +1,34 @@
 import 'babel-polyfill'
 import React from 'react'
-import { render } from 'react-dom'
+import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
-import App from './containers/App'
 import configureStore from './store/configureStore'
+import App from './containers/App'
 
 const store = configureStore()
+const rootEl = document.getElementById('root')
+
+function render(rootReactEl) {
+  ReactDOM.render(
+    <Provider store={store}>
+      {rootReactEl}
+    </Provider>,
+    rootEl
+  )
+}
```

Comment:
Why not put the try-catch here and call `render()` below so even if you reload and get a redbox it will still update with new changes?